README.md

```diff
@@ -277,7 +277,7 @@ async function fetchProductDetails(productId: string): Promise<Product> {
 
 const productId = 'product-123456'
 
-const result = await swr<Product>(productId, async () =>
+const result = await swr<Promise<Product>>(productId, async () =>
```

Comment:
Is it necessary for this type to be wrapped in a `Promise` here? Ideally, I would like to avoid that and have this generic type be the base/awaited type.

Reply:
Sure, I can fix that. Just need to update how the generic is used.